packages/client/base/src/types/embed/v3/CrossmintEmbeddedCheckoutV3Props.ts

```diff
@@ -0,0 +1,46 @@
+import { Currency as FiatCurrency, Locale } from "@/types";
+
+import { Blockchain } from "@crossmint/common-sdk-base";
+
+export interface CrossmintEmbeddedCheckoutV3Props {
+    recipient?: EmbeddedCheckoutV3Recipient;
+    locale?: Locale;
+    webhookPassthroughData?: any;
+    appearance?: EmbeddedCheckoutV3Appearance;
+    lineItems: EmbeddedCheckoutV3LineItem | EmbeddedCheckoutV3LineItem[];
+    payment: EmbeddedCheckoutV3Payment;
+}
+
+export type EmbeddedCheckoutV3Recipient = { email: string } | { walletAddress: string };
+
+export type EmbeddedCheckoutV3LineItem = {
+    collectionLocator: string;
+    callData?: Record<string, any>;
+};
+
+export type EmbeddedCheckoutV3Appearance = {
+    variables?: Record<string, string>; // TODO: Actual implementation
+};
+
+export type EmbeddedCheckoutV3Payment = {
+    fiat?: EmbeddedCheckoutV3FiatPayment;
+    crypto?: EmbeddedCheckoutV3CryptoPayment;
+};
```

Comment:
Would it make sense for this to be one OR the other? Similar to the recipient structure?